### PR TITLE
Bump branch alias

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.8.x-dev"
+            "dev-master": "1.9.x-dev"
```
Comment (mikeSimonson) [inline composer.json]: Shouldn't that be 1.10 now that we have a 1.9.1 ?

Comment (alcaeus) [reply composer.json]: When we have a support branch for 1.9, `dev-master` should be 1.10.

Comment (stof) [reply composer.json]: Do we actually need to maintain multiple branches in parallel ? Can't we just have master, and then deciding whether the next release is 1.9.2 or 1.10 based on what it contains (according to semver) ? Maintaining multiple branches creates more work.

Comment (alcaeus) [reply composer.json]: Absolutely - would you still want to bump branch-alias in that case? I'm not sure if it should be done preemptively or only after changes requiring a new minor release have been pushed to the branch.

Comment (stof) [reply composer.json]: I would keep it to 1.9.x then, until we have minor-level changes

**Comment by alcaeus** [comment]
Thanks @XWB!

**Pull request merged**